server/grpc_service.go

```diff
@@ -1410,8 +1410,8 @@ func (s *GrpcServer) GetOperator(ctx context.Context, request *pdpb.GetOperatorR
 	return &pdpb.GetOperatorResponse{
 		Header:   s.header(),
 		RegionId: requestID,
-		Desc:     []byte(r.Op.Desc()),
-		Kind:     []byte(r.Op.Kind().String()),
+		Desc:     []byte(r.Desc()),
+		Kind:     []byte(r.Kind().String()),
```

Comment:
Will this cause a compatibility problem?

Reply:
No, the `OperatorWithStatus` has inheritde `Operator` rather than one property. So the method of `Kind` returns some results.